universal-application-tool-0.0.1/app/services/program/ExportDefinition.java

```diff
@@ -22,6 +22,10 @@
   @Nullable
```

Comment:
(blocking): can these be made optional? 

Doesn't have to be part of this PR since it isn't part of this PR, but at least add a TODO.

Reply:
Sure, can do.